addons/languages/german/pack/src/main/res/xml/de_qwerty.xml

```diff
@@ -5,15 +5,15 @@
     android:keyWidth="10%p">
     
     <Row android:keyWidth="9.09%p">
-        <Key android:codes="113" android:keyLabel="q" android:popupCharacters="\@1" android:keyEdgeFlags="left"/>
+        <Key android:codes="113" android:keyLabel="q" android:popupCharacters="1\@" android:keyEdgeFlags="left"/>
         <Key android:codes="119" android:keyLabel="w" android:popupCharacters="2"/>
-        <Key android:codes="e"   android:keyLabel="e" android:popupKeyboard="@xml/de_popup_e" ask:hintLabel="€3"/>
+        <Key android:codes="e"   android:keyLabel="e" android:popupKeyboard="3@xml/de_popup_e" ask:hintLabel="€3"/>
```

Comment:
this can not be done: you can not add characters to `popupKeyboard` reference.
If you want to add the digit `3`, you need to edit the file `res/xml/de_popup_e.xml`

Reply:
I'll let this lie until https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2583 is (more) resolved